src/components/Avatar/Avatar.tsx

```diff
@@ -31,7 +31,7 @@ export type Props = {
 };
 
 export type AvatarSizes = 'xxxs' | 'xxs' | 'xs' | 'sm' | 'md' | 'lg';
-export type AvatarShapes = 'regular' | 'rounded' | 'circular';
+export type AvatarShapes = 'circular';
```

Comment:
is AvatarShapes type, and shape prop needed?

Reply:
As discussed off line I will not remove this in order to avoid a breaking change. They will be removed in v5 of Ictinus. 

Marking this as resolved.